CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Version 2.1.1](https://github.com/dataiku/dss-plugin-multisheet-excel-export/releases/tag/v2.1.1) - Major release - 2025-02
+
+- Add customization of tab names
+
```

Comment:
I'll rather bump to 2.2.0, that is a fairly new cool and important feature.

Reply:
done in [e75c529](https://github.com/dataiku/dss-plugin-multisheet-excel-export/pull/14/commits/e75c52966b4e8b5a2784f9780f6a2d3825b472de)